src/content/self-hosted/manage/troubleshooting.mdx

```diff
@@ -5,6 +5,68 @@ sidebar_label: Troubleshooting
 id: troubleshooting
 ---
 
+
+Welcome to the Okteto troubleshooting guide. This page provides answers to some common issues encountered while using Okteto.
+Please also review our [FAQ Guide](reference/faqs.mdx) for additional help.
+
+## How to extract logs from Okteto when asking for help
+
+### For Developers:
+When reaching out to Okteto for support or when asking the [Okteto Community](http://community.okteto.com), run [`okteto doctor` to generate a doctor file](reference/okteto-cli.mdx#doctor) with the okteto logs for a given development container.
+
+### For Administrators of Okteto:
+Please use our [Okteto Diagnostics tool](self-hosted/manage/diagnostics.mdx) to create a support bundle with cluster information, logs for okteto components, and other relevant information.
+
+## Understanding the state of your app: why [endpoints](core/endpoints/automatic-ssl.mdx) aren't present in the CLI or UI
```

Comment:
This page is for troubleshooting SH installations, that's why it's inside `src/content/self-hosted/manage`. 
But `why [endpoints](core/endpoints/automatic-ssl.mdx) aren't present in the CLI or UI` is much more general than that, it applies to developers and SaaS(BYOC customers too.
One option is to move the page to another place, like the left menu "Community and support".

Reply:
Ahh I see the issue. How about I move it over to the [FAQ page](https://www.okteto.com/docs/reference/faqs/)?

Reply:
That works too